config/config.go

```diff
@@ -421,6 +421,16 @@ func (r *Route) UnmarshalYAML(unmarshal func(interface{}) error) error {
 		groupBy[ln] = struct{}{}
 	}
 
+	if r.GroupWait != nil && time.Duration(*r.GroupWait) == time.Duration(0) {
```

Comment:
A group_wait of 0 makes sense (though it's a bad idea).

Reply:
Right, I was too hasty...